nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core-api/src/main/java/org/apache/nifi/controller/AbstractConfiguredComponent.java

```diff
@@ -57,6 +57,7 @@ public AbstractConfiguredComponent(final ConfigurableComponent component, final
         this.component = component;
         this.validationContextFactory = validationContextFactory;
         this.serviceProvider = serviceProvider;
+        this.name = new AtomicReference<>(component.getClass().getSimpleName());
```

Comment:
My concern is it looks like quite a few things inherit from AbstractConfiguredComponent, and now every one of them has changed, not just StandardProcessorNode